pg/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  postgres:
+    image: postgres:9.6
```

Comment:
Did this affect the performance of our benchmark? We'll probably have to invalidate all benchmarks that were ran with Postgres 9.5